addon/components/ember-loadify/pagination.js

```diff
@@ -25,10 +28,44 @@ export default Component.extend(InViewportMixin, {
     }
   },
 
-  pageLinks: computed('totalPages', function() {
-    return Array.from({length: this.get('totalPages')}, (v, k) => k+1);
+  nextPages: computed('currentPage', 'isFirstPage', function() {
+    const startPage = (this.isFirstPage ? 2 : this.currentPage);
+    return this._range(startPage, (this.currentPage + 5));
```

Comment:
return this._range(this.currentPage, endPage)

Reply:
also,

I think here you should

psuedo code
```
return this._range().delete(currentPage)
```

But make sure it still returns an array. It would essentially return an empty array if there are no pages after current, ex: you're on the last page